src/setup.py

```diff
@@ -7,15 +7,16 @@
 print(__file__)
 print(MAIN_DIR)
 # The text of the README file
-README = open(os.path.join(MAIN_DIR, "README.md")).read()
-
+# README = open(os.path.join(MAIN_DIR, "README.md")).read()
```

Comment:
shouldn't remove this commented lines?

Reply:
Oh yea I forgot about that. There is some issue with `tox` with packing the README. Tox works by taking your code and creating it as a package (which is fine because it is already a package), but the way it does the packaging makes the top level directory not available. This causes an error because it can't find the file.

You should be able to reproduce this by uncommenting the line then running the unit tests. 

I'll wrap this in a quick function that returns a static string if the file is not available, which should stop gap this issues. (As long as in the real packaging the project readme correctly gets read this shouldn't be a big deal)